tests/test_schema_text.py

```diff
@@ -0,0 +1,50 @@
+import os
+from json import load
+
+from structlog import configure
+from structlog import getLogger
+from structlog.stdlib import LoggerFactory
+
+from app.validation.validate_schema_text import validate_json_schema_text
+
+logger = getLogger()
+
+configure(logger_factory=LoggerFactory())
```

Comment:
Is the logger configuration needed? I don't see any log lines in the below code

Reply:
I have removed these